File: exercicios/cpp/Instruções Sequenciais/Q21.cpp
#include <iostream>
#include <clocale>
#include <cmath>

using namespace std;

int main() {
	setlocale(LC_ALL,"Portuguese");
    float imc, peso, altura;
    count<<"Informe o peso (kg) e a altura(m):";
    cin>>peso>>altura;
    imc = peso/ pow(altura, 2);
    cout<<"IMC:"<<imc<<endl;
    return 0;
}

File: exercicios/python/Instruções Sequenciais/Q21.py
# Exercício 21 — Cálculo de IMC
def main():
    nome = input("Nome: ")
    peso = float(input("Peso (kg): "))
    altura = float(input("Altura (m): "))
    imc = peso / (altura**2)
    print(f"{nome} — IMC: {imc:.2f}")

if __name__ == "__main__":
    main()